var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -289,12 +320,28 @@ def write_library_dirs(f, dirs):
                         )
                     )
 
+    def setup_dependent_build_environment(self, env, depspec):
+        self.setup_build_environment(env)
+
     def setup_build_environment(self, env):
         # Tell numpy which BLAS/LAPACK libraries we want to use.
         # https://github.com/numpy/numpy/pull/13132
         # https://numpy.org/devdocs/user/building.html#accelerated-blas-lapack-libraries
         spec = self.spec
 
+        if spec['blas'].name != 'intel-mkl' and \
+                spec['blas'].name != 'intel-parallel-studio':
```

Comment:
Apparently `intel-oneapi-mkl` also provides MKL now. Can you update this here and below?

Reply:
yep